### DISCUSSION TITLE
studiekeuzecheck

### DISCUSSION THREAD

**Discussion by iBadr49** [fdnd/docs.fdnd.nl#85] [Reflectie]
Als student heb ik twee keer deelgenomen aan de open dag van mijn opleiding en heb ik geholpen bij het beantwoorden van vragen van andere studenten. Tijdens deze ervaringen viel het me op dat veel studenten soms twijfelen of de opleiding echt bij hen past, zelfs als ze weten dat ze bijvoorbeeld websites gaan bouwen met verschillende frameworks. Ik geloof dat een studiekeuze check (optioneel) een waardevolle manier is om te ontdekken of een opleiding echt bij je past. Dit kun je doen door een opdracht te maken en zelf uit te proberen, zodat je een beter gevoel krijgt of de studie aansluit bij jouw interesses en vaardigheden.

**Comment by ju5tu5** [comment]
Opgenomen in de tekst
